pages/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Inter } from "next/font/google";
+
+const inter = Inter({ subsets: ["latin"] });
```

Comment:
Move these into a separate `fonts` file

Reply:
@motuncoded Can you make sure to resolve conversations when you have "fixed" them so that this page is cleaner to read and we can easily identify whatever is left unfixed

Reply:
Ok. Noted.